backend/src/main/java/hanglog/global/BaseEntity.java

```diff
@@ -45,4 +45,8 @@ public boolean isDeleted() {
     public void changeStatusToDeleted() {
         this.status = DELETED;
     }
+
+    public void changeStatusToUsable() {
+        this.status = USABLE;
+    }
```

Comment:
이렇게는 어떨지 ~~ ? (제안제안제안)
물론 메서드 명으로 명확하게 볼 수 있어서 이것도 좋아요
그런뒈 이거 사용안하는거 같은데 그냥 만들어두신건가요 ?!

```suggestion
    public void changeStatus(final StatusType status) {
        this.status = status;
    }
```

Reply:
ㅎ.ㅎ 사용했다가 필요없어진 기능인데, 지우겠습니 다